.Gruntfile.babel.js

```diff
@@ -112,7 +112,7 @@ module.exports = (grunt) => {
     exec: {
       // could replace w/https://github.com/pghalliday/grunt-mocha-test
       test: {
-        cmd: './node_modules/.bin/mocha --require Gruntfile.js -R spec --bail',
+        cmd: './node_modules/.bin/mocha --require Gruntfile.js -R spec --bail "{./{,!(node_modules)/**/}*.test.js,./test/*.js}"',
```

Comment:
Anything not instantly understandable (which this definitely qualifies) should have a comment above it to explain what's going on / why this is being done. This saves developers headache and you time (because they won't have to figure out who to ask)